roles/puppet_repositories/tasks/puppetlabs-6-redhat.yml

```
@@ -0,0 +1,6 @@
+---
+- name: 'Setup Puppet 6 Repository'
+  yum:
+    name: https://yum.puppet.com/puppet6-release-el-{{ ansible_distribution_major_version }}.noarch.rpm
```

I wonder if we need a versioned file these days, or if that can be:
```suggestion
    name: https://yum.puppet.com/puppet{{ puppet_repositories_version }}-release-el-{{ ansible_distribution_major_version }}.noarch.rpm
```